lib/mix/tasks/hex.owner.ex

```diff
@@ -79,6 +91,7 @@ defmodule Mix.Tasks.Hex.Owner do
   def tasks() do
     [
       {"add PACKAGE EMAIL_OR_USERNAME", "Adds an owner to package"},
+      {"transfer PACKAGE EMAIL_OR_USERNAME", "Transfers ownership of a package to another user"},
```

Comment:
Even though an organisation corresponds to a user, wdyt about mentioning both in docs?

```suggestion
      {"transfer PACKAGE EMAIL_OR_USERNAME", "Transfers ownership of a package to another user or organization"},
```